demo/src/main/java/com/example/demo/controller/PeopleController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.People;
+import com.example.demo.repository.PeopleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class PeopleController {
+
+    @Autowired
+    private PeopleRepository peopleRepository;
```

Comment:
please don't directly autowire repo in the controller

Reply:
this could be `final` and you could use @RequiredArgsConstructor to autowire (field injection is not recommended)

Reply:
this comment applies everywhere where you autowire stuff